app/src/main/res/values/strings.xml

```diff
@@ -86,7 +86,7 @@
     <string name="app_libraries">Libre third-party libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
     <string name="app_resources">Libre third-party resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
     <string name="selectBarcodeTitle">Select Barcode</string>
-    <string name="enterBarcodeInstructions">Enter the ID, and either pick a barcode type below, or “There is no barcode”.</string>
+    <string name="enterBarcodeInstructions">Enter the ID number or text on your card and press the barcode that looks like the one on your card.</string>
```

Comment:
Outdated translations might still continue to refer to "There is no barcode" which no longer exists. Seems potentially confusing. But replacing the string with another key and thus have no translation at all doesn't seem much better either. Just pointing it out.